release-notes/v2.4.1.md

```diff
@@ -0,0 +1,33 @@
+# HPE CSI Driver for Kubernetes 2.4.1 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.4.1
+
+| **Version:** |2.4.1|
+|--------------|-----|
+| **Revision:** | Tuesday January 30, 2024 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.4.1.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|CON-3005|csi.k8s|RWX support for raw block volumes in Primera||
+|CON-3027|csi.k8s|ARM64 images support for the Nimble Container Storage Provider||
+|CON-3052|csi.k8s|Enumerate capabilities and volume types in OpenShift Security Context Constraints| Updated the SCOD documenatation|
+|CON-3020|csi.k8s|3PAR Conatiner Storage Provider unable to figure out the least used CPG for Alletra MP||
+|CON-3012|csi.k8s|Support Alletra Storage MP with 3PAR CSP||
+|CON-3009|csi.k8s|ARM64 images support for the CSI driver||
+|CON-1743|csi.k8s|Support foreign StorageClass for multi-writers|Made enhancement to handle `nfsForeignStorageClass` parameter in storage class|
+|CON-3042|csi.k8s|Support and conform to SLE Micro||
+|CON-3044|csi.k8s|Removed glide(a package managing utility in GoLang) in common-host-libs and common-host-utils|Use go.mod to manage dependencies|
+
```

Comment:
Please include the issue. CON-3072  also, the fix for Arcus R3 connectivity issue